src/prefix_tf_republisher.cpp

```diff
@@ -0,0 +1,11 @@
+#include <hector_multi_robot_tools/prefix_tf_republisher_node.hpp>
+
+int main(int argc, char **argv)
+{
+    rclcpp::init(argc, argv);
+
+    const auto node = std::make_shared<hector_multi_robot_tools::PrefixTfRepublisherNode>();
+    spin(node);
```

Comment:
Why does this not need to be qualified as `rclcpp::spin`?

Reply:
No idea, I will add it again just to be sure.